src/core/optimization_container.jl

```diff
@@ -319,14 +348,39 @@ function build_impl!(container::OptimizationContainer, template, sys::PSY.System
     end
 
     for branch_model in values(template.branches)
```

Comment:
We probably don't need to separate Branches anymore since the Arguments are built before the network. We could simplify the code further. @claytonpbarrows we should revisit the PM interface in a later PR to simplify the code.

Reply:
This will be resolved at a later stage.